### PR TITLE
chore(docs): Adding OpenCensus integration instructions

### PR DESCRIPTION
**Pull request by shantstepanian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,206 @@ Note: Please use [google-cloud-bigtable][google-cloud-bigtable] to access Bigtab
...
+import com.google.bigtable.repackaged.io.opencensus.trace.Tracing;
+import com.google.bigtable.repackaged.io.opencensus.trace.samplers.Samplers;
+
+import java.io.IOException;
+
+public class OpenCensusExample {
+    String projectId = "your-project-id";
+
+    void setupTracing() throws Exception {
+        //// Setup tracing.
```
Comment (mutianf) [inline README.md:249]: nit: // 😅

File: README.md [hunk 2]
```diff
@@ -62,6 +62,206 @@ Note: Please use [google-cloud-bigtable][google-cloud-bigtable] to access Bigtab
...
+        // Use the code snippet below as a starting example.
+        // For examples of monitored resources, see here: https://cloud.google.com/monitoring/api/resources
+        StackdriverStatsExporter.createAndRegister(
+                StackdriverStatsConfiguration.builder()
+                        .setProjectId(projectId)
+                        // This example uses generic_node as the MonitoredResource, with your host name as the node ID.
+                        .setMonitoredResource(MonitoredResource.newBuilder()
+                                .setType("generic_node")
+                                .putLabels("project_id", projectId)
+                                .putLabels("location", "us-west1-b")
```
Comment (mutianf) [inline README.md:157]: Should we also add a comment here, something like "The region in which the service is running, in this case us-west1-b"?

File: README.md (hunk 1 above)
Comment (shantstepanian) [reply README.md]: Fixed

File: README.md [hunk 3]
```diff
@@ -62,6 +62,194 @@ Note: Please use [google-cloud-bigtable][google-cloud-bigtable] to access Bigtab
...
+
+If you are _not_ using the shaded Bigtable HBase Client artifact, you need to define the OpenCensus dependencies.
+
+[//]: # ({x-version-update-start:bigtable-hbase-1x-parent:released})
+```xml
+<!-- OpenCensus dependencies -->
+<dependency>
+    <groupId>com.google.cloud.bigtable</groupId>
+    <artifactId>bigtable-hbase-1.x</artifactId>
+    <version>1.14.1</version>
```
[the remark this answers is not in the document]
Comment (shantstepanian) [reply README.md]: Done - all references to 1.14.1 have been updated to 1.18.0

File: README.md (hunk 2 above)
Comment (shantstepanian) [reply README.md]: Done

**Review by kolea2 — APPROVED** [review]

**Review by mutianf — APPROVED** [review]
lgtm!

**Review by igorbernstein2 — APPROVED** [review]
lgtm

**Pull request merged**